### PR TITLE
Add properties field to Permit model

### PR DESCRIPTION
**Pull request by amanpdyadav**
Permit model requires non-functional field to denote
the type of permit. And as per the discussion it
was decided to use it as a property in the meta field.

meta = {
   'permit_type': ''Asukaspysäköintitunnus
}

Refs PV-342

### CODE REVIEW

**Comment by amanpdyadav** [comment]
> Please restore the contents from your old commit ([85d4ae8](https://github.com/City-of-Helsinki/parkkihubi/commit/85d4ae88128db1114237ad8a74de951de17b2819) or [a13b488](https://github.com/City-of-Helsinki/parkkihubi/commit/a13b4883858e12db90c9ab944571b1c9092e26df) probably?) and rebase on top of master so we can see that tests pass.

Thanks for pointing it out. I had accidentally force pushed other changes to it. But now it should be Ok.

File: parkings/models/permit.py [hunk 1]
```diff
@@ -112,6 +114,8 @@ class Permit(TimestampedModelMixin, models.Model):
         'end_time': TimestampField(),
         'area': TextField(max_length=10),
     })])
+    # Meta
+    meta = JSONField(blank=True, null=True, encoder=DjangoJSONEncoder, verbose_name=_("meta"))
```
Comment (suutari-ai) [inline parkings/models/permit.py:118]: The comment doesn't add any value. Better just remove it.

File: parkings/tests/api/operator/test_permit.py [hunk 2]
```diff
@@ -26,6 +26,7 @@ def _check_response(data, obj):
         'id',
         'subjects',
         'areas',
+        'meta'
```
Comment (suutari-ai) [inline parkings/tests/api/operator/test_permit.py:29]:
```suggestion
        'meta',
```

File: parkings/models/permit.py (hunk 1 above)
Comment (suutari-ai) [reply parkings/models/permit.py:118]:
```suggestion
    meta = JSONField(blank=True, null=True, encoder=DjangoJSONEncoder, verbose_name=_("meta"))
```

File: parkings/tests/api/operator/test_permit.py [hunk 3]
```diff
@@ -204,6 +205,7 @@ def test_area_restriction(
         'subjects': generate_subjects(),
         'areas': [area],
         'domain': 'HKI',
+        'meta': {'permit_type': 'Asukaspysäköintitunnus'}
```
Comment (suutari-ai) [inline parkings/tests/api/operator/test_permit.py:208]:
```suggestion
        'meta': {'permit_type': 'Asukaspysäköintitunnus'},
```

File: parkings/models/permit.py (hunk 1 above)
Comment (amanpdyadav) [reply parkings/models/permit.py]: Done

File: parkings/tests/api/operator/test_permit.py (hunk 2 above)
Comment (amanpdyadav) [reply parkings/tests/api/operator/test_permit.py]: Done

File: parkings/tests/api/operator/test_permit.py (hunk 3 above)
Comment (amanpdyadav) [reply parkings/tests/api/operator/test_permit.py]: Done

File: parkings/api/common_permit.py [hunk 4]
```diff
@@ -78,6 +78,7 @@ class Meta:
             'external_id',
             'subjects',
             'areas',
+            'meta'
```
[the remark this answers is not in the document]
Comment (amanpdyadav) [reply parkings/api/common_permit.py]: Done

**Review by suutari-ai — APPROVED** [review]
Thanks for the fixes! Looks good now! :+1:

**Pull request merged**